kie-wb-common-react/boxed-expression-editor/src/components/ExpressionContainer/ExpressionContainer.css

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#expression-container-box {
```

Comment:
I believe we should continue keeping our custom CSS with a parent CSS class for each component, that helps a lot with "leaks" :-) 

I believe we already have the `.expression-container` CSS class:

<img width="509" alt="image" src="https://user-images.githubusercontent.com/1079279/99454813-b4cf6c80-2905-11eb-9442-4c1690c5b86d.png">

So a believe it's just a matter of using it like this:
```suggestion
.expression-container #expression-container-box {
```